make/common/native/Link.gmk

```diff
@@ -109,6 +109,11 @@ define CreateStaticLibrary
 	      $(if $$($1_LINK_OBJS_RELATIVE), $$(CD) $$(OUTPUTDIR) ; ) \
 	      $$($1_LD) $(LDFLAGS_CXX_PARTIAL_LINKING) $$($1_SYSROOT_LDFLAGS) \
 	          -o $$($1_TARGET_RELOCATABLE) $$($1_LD_OBJ_ARG))
+          # Linking with '-Wl,--icf=safe' using objects created by 'ld -r' may not work
+          # and could cause errors like the following:
+          #   ld: --icf=safe conservatively ignores SHT_LLVM_ADDRSIG [...] with sh_link=0 (likely created using objcopy or ld -r)
```

Comment:
I'd recommend rephrasing the comment somewhat, like

```suggestion
          # 'ld -r' might fail to update the .llvm_addrsig section, and this will cause subsequent
          # calls to lld to fail when linking with this library, so remove that section.
```

(adjust line lengths if > 80)

Reply:
Suggestion applied, with some minor additional adjustments. Thanks.